bitnami/zookeeper/templates/scripts-configmap.yaml

```diff
@@ -31,6 +34,7 @@ data:
               -keystore "/opt/bitnami/zookeeper/config/certs/client/zookeeper.truststore.jks" \
               -storepass "${ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD}" \
               -noprompt
+        echo "1" > /opt/bitnami/zookeeper/config/certs/client/initialized
```

Comment:
I'd create a hidden file without any content, this way the file won't be listed by default so it'll create less noise. What do you think?

```
touch /opt/bitnami/zookeeper/config/certs/client/.initialized
```

Reply:
Done